forge-gui/res/cardsfolder/upcoming/mephidross_slime.txt

```diff
@@ -0,0 +1,18 @@
+Name:Mephidross Slime
+ManaCost:1 B G
+Types:Creature Phyrexian Ooze
+PT:3/3
+K:Trample
+T:Mode$ ChangesZone | ValidCard$ Card.Self | Origin$ Battlefield | Destination$ Graveyard  | Execute$ TrigConjure | Secondary$ True | TriggerDescription$ Whenever CARDNAME dies ,conjure a card named Mephidross Slime into your graveyard. Perpetually double the power and toughness of CARDNAME and that card. then shuffle them into their owner's library.
+SVar:TrigConjure:DB$ MakeCard | Conjure$ True | Name$ Mephidross Slime | Amount$ 1 | Zone$ Library | RememberMade$ True | NumAtt$ +X | Double$ True | SubAbility$ DoubleMade
```

Comment:
Also not sure if you think those Pump params can work there or just copy/paste error

Reply:
Yeah copy paste error